ts/config/highcharts-config.d.ts

```diff
@@ -0,0 +1,147 @@
+import 'highcharts/ts/masters/indicators/bollinger-bands.src';
+import 'highcharts/ts/masters/indicators/cci.src';
+import 'highcharts/ts/masters/indicators/chaikin.src';
+import 'highcharts/ts/masters/indicators/cmf.src';
+import 'highcharts/ts/masters/indicators/cmo.src';
+import 'highcharts/ts/masters/indicators/dema.src';
+import 'highcharts/ts/masters/indicators/disparity-index.src';
+import 'highcharts/ts/masters/indicators/dmi.src';
+import 'highcharts/ts/masters/indicators/dpo.src';
+import 'highcharts/ts/masters/indicators/ema.src';
+import 'highcharts/ts/masters/indicators/ichimoku-kinko-hyo.src';
+import 'highcharts/ts/masters/indicators/indicators-all.src';
+import 'highcharts/ts/masters/indicators/indicators.src';
+import 'highcharts/ts/masters/indicators/keltner-channels.src';
+import 'highcharts/ts/masters/indicators/klinger.src';
+import 'highcharts/ts/masters/indicators/macd.src';
+import 'highcharts/ts/masters/indicators/mfi.src';
+import 'highcharts/ts/masters/indicators/momentum.src';
+import 'highcharts/ts/masters/indicators/natr.src';
+import 'highcharts/ts/masters/indicators/obv.src';
+import 'highcharts/ts/masters/indicators/pivot-points.src';
+import 'highcharts/ts/masters/indicators/ppo.src';
+import 'highcharts/ts/masters/indicators/price-channel.src';
+import 'highcharts/ts/masters/indicators/price-envelopes.src';
+import 'highcharts/ts/masters/indicators/psar.src';
+import 'highcharts/ts/masters/indicators/regressions.src';
+import 'highcharts/ts/masters/indicators/roc.src';
+import 'highcharts/ts/masters/indicators/rsi.src';
+import 'highcharts/ts/masters/indicators/slow-stochastic.src';
+import 'highcharts/ts/masters/indicators/stochastic.src';
+import 'highcharts/ts/masters/indicators/supertrend.src';
+import 'highcharts/ts/masters/indicators/tema.src';
+import 'highcharts/ts/masters/indicators/trendline.src';
+import 'highcharts/ts/masters/indicators/trix.src';
+import 'highcharts/ts/masters/indicators/volume-by-price.src';
+import 'highcharts/ts/masters/indicators/vwap.src';
+import 'highcharts/ts/masters/indicators/williams-r.src';
+import 'highcharts/ts/masters/indicators/wma.src';
+import 'highcharts/ts/masters/indicators/zigzag.src';
+
+import 'highcharts/ts/masters/themes/avocado.src';
+import 'highcharts/ts/masters/themes/brand-dark.src';
+import 'highcharts/ts/masters/themes/brand-light.src';
+import 'highcharts/ts/masters/themes/dark-blue.src';
+import 'highcharts/ts/masters/themes/dark-green.src';
+import 'highcharts/ts/masters/themes/dark-unica.src';
+import 'highcharts/ts/masters/themes/gray.src';
+import 'highcharts/ts/masters/themes/grid-light.src';
+import 'highcharts/ts/masters/themes/grid.src';
+import 'highcharts/ts/masters/themes/high-contrast-dark.src';
+import 'highcharts/ts/masters/themes/high-contrast-light.src';
+import 'highcharts/ts/masters/themes/sand-signika.src';
+import 'highcharts/ts/masters/themes/skies.src';
+import 'highcharts/ts/masters/themes/sunset.src';
```

Comment:
Do we still need these imports?

Reply:
Yes, this config file is still needed. We could limit the number of these imports, but to keep it safe, I'd leave them as they are now.

Reply:
Hmm, where it is used?

Reply:
`"include": ["ts/**/*"]`

We're taking all the paths in the `ts` folder for the compilation, so this config is taken as well and prevents the HC errors. If you try to remove these imports and compile the code, it will break with numerous errors. 🤔

Reply:
I see! Well, thanks for the clarification!